packages/oui-popover/src/popover.controller.js

```diff
@@ -119,6 +138,10 @@ export default class PopoverController {
 
         // Support for attribute `oui-popover`
         this.$element.attr("aria-expanded", true);
+
+        if (angular.isDefined(this.onOpen)) {
+            this.$timeout(() => this.onOpen(), 0);
```

Comment:
Why use a `$timeout` for the callbacks ?
Is it not working without it, like the others ?

If not, it could be a nice add to have a comment about it ;)